Packetbroker agent flaky test

<!--
Thanks for submitting a bug report. Please fill the template below,
otherwise we will not be able to process this bug report.
-->

#### Summary
<!-- Summarize the problem in a few sentences: -->

`TestForwarder` test seems to be flaky.

#### Steps to Reproduce
<!-- How can we reproduce the problem? -->

bad luck I guess 🤷‍♂️

<!--
Please upload relevant configuration (as .txt).
If you use the command "ttn-lw-stack config", you can redact sensitive config.
-->

#### What do you see now?
<!-- Please paste terminal output, upload logs (as .txt) or upload screenshots. -->

```
 --- FAIL: TestForwarder (0.38s)
    --- FAIL: TestForwarder/Uplink (0.13s)
        --- FAIL: TestForwarder/Uplink/0 (0.13s)
            agent_test.go:314: Expected uplink message from Forwarder
FAIL
coverage: 11.1% of statements
FAIL    go.thethings.network/lorawan-stack/v3/pkg/packetbrokeragent     0.438s
```

#### What do you want to see instead?
<!-- Please add some examples or mock-ups if applicable. -->

The test should not accidentally fail and then pass with no changes.

#### Environment
<!--
Your environment: OS/Browser/Gateway/Device/...? Versions? IDs/EUIs?
Paste the output of "ttn-lw-cli version" or "ttn-lw-stack version" if applicable.
-->

v3.9.1


#### Can you do this yourself and submit a Pull Request?
<!-- You can also @mention experts if you need help with this. -->

nope

--- a/pkg/packetbrokeragent/agent_test.go
+++ b/pkg/packetbrokeragent/agent_test.go
@@ -43,7 +43,7 @@ import (
 )
 
 var (
-	timeout     = (1 << 4) * test.Delay
+	timeout     = (1 << 5) * test.Delay
 	testOptions []Option
 )
 

